lib/server-starter.js

```diff
@@ -90,7 +92,23 @@ class Server extends EventEmitter {
     });
 
     // Should be resolved by the "spawn" event, but that is only supported in Node 15+
-    return Promise.resolve;
+    if (process.platform !== 'win32') return Promise.resolve;
+    return new Promise((resolve) => {
+      const retryTime = 60;
+      const now = new Date();
```

Comment:
This constant does nothing.

Reply:
it is used in the following line to calculate timeToStop;

Reply:
But there is no use in giving it a name.

Reply:
`const timeToStop = new Date(Date.now() + win32Timeout);`?

Reply:
Ok